source/compatibility.tex

```diff
@@ -2004,6 +2004,35 @@
 };
 \end{codeblock}
 
+\diffref{class.copy.elision}
+\change
+A function returning an implicitly movable entity may invoke
+a constructor taking an rvalue reference to a type
+different from that of the returned expression.
+Function and catch-clause parameters can be thrown using move constructors.
+\rationale
+Side-effect of making it easier to write
```

Comment:
Not a complete sentence, and we don't hyphenate side-effects.  Maybe "Side-effect" -> "Has the side effect of"?

Reply:
We don't do complete sentences for \rationale; cf. "Necessary for new functionality."

Removed hyphen from "Side-effect".